Client/Frontend/Strings.swift

```diff
@@ -347,6 +347,54 @@ extension String {
                 comment: "Button title which, when tapped, will allow the user to save valid credit card details.")
         }
 
+        // Remember Card
+        public struct RememberCard {
+            public static let MainTitle = MZLocalizedString(
+                key: "RememberCardTitle.Default",
+                tableName: nil,
```

Comment:
All tablename shouldn't be `nil`, as per documentation we don't know when as string can become experiementable, and tablename is needed for experimentable strings.
```suggestion
                tableName: "RememberCard",
```

Similar comment will apply for strings under `UpdateCard`